src/topology/monitor.rs

```diff
@@ -10,7 +10,7 @@ use command_type::CommandType;
 use connstring::{self, Host};
 use cursor::Cursor;
 use pool::ConnectionPool;
-use stream::StreamConnector;
+use stream::ConnectMethod;
```

Comment:
Changes in this file are a mix of the renaming of `StreamConnector` and from rustfmt.